discord/utils.py

```diff
@@ -1027,3 +1029,53 @@ def format_dt(dt: datetime.datetime, /, style: Optional[TimestampStyle] = None)
     if style is None:
         return f'<t:{int(dt.timestamp())}>'
     return f'<t:{int(dt.timestamp())}:{style}>'
+
+
+def basic_autocomplete(values: Union[Iterable[str],
```

Comment:
```suggestion
def basic_autocomplete(values: Union[Iterable[str],
                                     Callable[[Interaction], Union[Iterable[str], Coroutine[Iterable[str]]]],
                                     Coroutine[Iterable[str]]]
) -> Callable[[Interaction, str], Coroutine[List[str]]]:
```
Maybe a better way of formatting it?

Reply:
Should be fine, its 2 characters short of 120 long.

Reply:
ah ok